splinterd/tests/admin/circuit_disband.rs

```diff
@@ -49,22 +50,33 @@ pub fn test_2_party_circuit_lifecycle() {
     let node_a = network.node(0).expect("Unable to get first node");
     // Get the second node in the network
     let node_b = network.node(1).expect("Unable to get second node");
-    let node_b_admin_pubkey = admin_pubkey(node_b);
-    let circuit_id = "ABCDE-01234";
     // Commit the circuit to state
-    commit_2_party_circuit(&circuit_id, node_a, node_b);
+    let node_b_admin_pubkey = admin_pubkey(node_b);
+
+    let CircuitData {
+        circuit_id,
+        management_type,
+        ..
+    } = network
+        .circuit_builder(&[0, 1])
+        .expect("Could not create builder")
+        .veil::<Scabbard>()
+        .add_service_group(&[0, 1])
+        .unwrap()
+        .build()
+        .expect("Could not create circuit");
 
     // As we've started a new event client, we'll skip just past the circuit ready event
     let mut node_a_events = BlockingAdminServiceEventIterator::new(
         node_a
-            .admin_service_event_client(&format!("test_circuit_{}", &circuit_id))
+            .admin_service_event_client(&management_type)
```

Comment:
Is this change intentional?

Reply:
Yeah, I was just taking advantage of the fact that the management type for the circuit is returned by the circuit builder.